### PR TITLE
Document memory autotuning strategy

### PR DESCRIPTION
**Pull request by crezvoy**
# Jira Ticket

[BGD-3987](https://spotinst.atlassian.net/browse/BGD-3987)

# Demo

Screenshot 2023-10-06 at 15 01 20

[BGD-3987]: https://spotinst.atlassian.net/browse/BGD-3987?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: api/services/ocean/spark/schemas/oceanSparkApplicationSpec.yaml [hunk 1]
```diff
@@ -1243,6 +1244,35 @@ components:
           type: string
       required:
       - strategy
+    MemoryAutotuningStrategy:
+      description: Memory allocation strategy that automatically increments the
+        memory when a previous app failed with a OOM, and decreases it based on
```
Comment (thorsteinnth) [inline api/services/ocean/spark/schemas/oceanSparkApplicationSpec.yaml:1249]: nit: `an OOM`

File: api/services/ocean/spark/schemas/oceanSparkApplicationSpec.yaml [hunk 2]
```diff
@@ -1243,6 +1244,35 @@ components:
...
+        past applications memory usage
```
Comment (thorsteinnth) [inline api/services/ocean/spark/schemas/oceanSparkApplicationSpec.yaml:1250]: nit: `applications'`

File: api/services/ocean/spark/schemas/oceanSparkApplicationSpec.yaml (hunk 1 above)
Comment (crezvoy) [reply api/services/ocean/spark/schemas/oceanSparkApplicationSpec.yaml:1249]: fixed

File: api/services/ocean/spark/schemas/oceanSparkApplicationSpec.yaml (hunk 2 above)
Comment (crezvoy) [reply api/services/ocean/spark/schemas/oceanSparkApplicationSpec.yaml]: fixed

**Review by thorsteinnth — APPROVED** [review]

**Pull request merged**